docker-compose/monitor/jaeger_example.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+# jaeger_example.py
+from opentelemetry import trace
+from opentelemetry.exporter.jaeger.thrift import JaegerExporter
+from opentelemetry.trace import SpanKind
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+from opentelemetry.sdk.resources import Resource
+
+resource = Resource(attributes={
+    "service.name": "my_service"
+})
+
+trace.set_tracer_provider(TracerProvider(resource=resource))
+
+jaeger_exporter = JaegerExporter(
+    collector_endpoint="http://localhost:14278/api/traces",
+)
```

Comment:
I would suggest using OTLP exporter instead and exposing OTLP port in the OTEL Collector.

Reply:
For my learning, why the preference for OTLP over Jaeger formatted spans?

Done in [ab3dd6b](https://github.com/jaegertracing/jaeger/pull/3996/commits/ab3dd6b6f5f42a6347ce7659f5c2da7b9e63e9f2).

Reply:
Because jaeger SDKs are deprecated, the recommended path is native OTEL